app.json

```diff
@@ -75,6 +75,12 @@
         "value": "",
         "required": false
     },
+    "WHOOGLE_CONFIG_DISABLE_CHANGE": {
```

Comment:
I think just `WHOOGLE_CONFIG_DISABLE` would be a better name, the fact that it disables changes specifically is implied (just my opinion, let me know if you disagree). 

If you think it should include the `CHANGE` qualifier, my only nitpick would be to update it to `WHOOGLE_CONFIG_DISABLE_CHANGES`.